Game/marvin.cpp

```diff
@@ -130,8 +136,38 @@ bool Marvin::exec(Gothic& gothic, const std::string& v) {
         c->setToogleEnable(!c->isToogleEnabled());
       return true;
       }
+    case C_Insert: {
+      World* world = gothic.world();
+      Npc* player = gothic.player();
+
+      std::string::size_type spacePos = v.find(" ");
+
+      if( spacePos != std::string::npos ) {
+        std::string arguments = v.substr(spacePos + 1);
+
+        if( world == nullptr || player == nullptr ) {
+          return false;
+        } else{
```

Comment:
no need to `else`, after return.

Reply:
I think you have read that incorrectly. return is in if statement. If the condition fails it goes to else without ever hitting return. I think code is correct.

Reply:
@AdamBilisic, yes this code is correct, but `else` operator still doesn't affect anything:
if( {true} ) - early-out, by return
if( {false} ) - just continue forward, without else

Reply:
Ah yeah so you meant redundant else because if already returns value out of the function. Yes this is personal preference but I like it too. So desired outcome is

 if( world == nullptr || player == nullptr ) {
    return false;
 }
 return addItemOrNpcBySymbolName(world, arguments, player->position());

Reply:
Sorry, had a lot todo the last few days.
Should be fixed now.